test/exercism/user_test.rb

```diff
@@ -103,6 +103,47 @@ def test_delete_team_memberships_with_user
     refute TeamMembership.exists?(team: other_team, user: bob, inviter: alice), 'Unconfirmed TeamMembership was deleted.'
   end
 
+  def test_non_qualifying_comment_does_not_add_to_table
+    alice = User.create(username: 'alice')
+    fred = User.create(username: 'fred')
+    submission = Submission.create(user: alice, language: 'python', slug: 'one')
+    submission.reload
+    comment = Comment.create(user: alice, submission: submission, body: 'something')
+    alice.increment_five_a_day if comment.qualifying?
```

Comment:
We don't need to worry about the qualifying/not qualifying, because we already have tests for this. All we need to test is what `increment_five_a_day` actually does.